src/java/fr/paris/lutece/portal/service/daemon/AppDaemonService.java

```diff
@@ -295,8 +295,9 @@ private static void scheduleThread( DaemonEntry entry )
      */
     private static void scheduleThread( DaemonEntry entry, int nInitialDelay )
     {
-        _executor.schedule( entry, nInitialDelay, TimeUnit.SECONDS );
+        AppLogService.info( "Scheduling daemon " + entry.getId( ) + "; first run in " + nInitialDelay + " seconds" );
         entry.setIsRunning( true );
+        _executor.schedule( entry, nInitialDelay, TimeUnit.SECONDS );
         // update onStartup property
```

Comment:
I explained in https://github.com/lutece-platform/lutece-core/pull/93#issuecomment-310894140
I think the daemon can start executing before isRunning is set, which causes it to unschedule itself.